### DISCUSSION TITLE
REMOTE DESKTOP PROTOCOL FAILURE. FreeRDP failed to establish a connection with Windows. Alpine Linux

### DISCUSSION THREAD

**Discussion by Rosnerv** [winapps-org/winapps#282] [Support]
"I have tried several versions of Alpine Linux (3.18; 3.20; Alpha), and none of them allow me to get WinApps working.

I have an installation with Docker, and it fails to connect when running the installer. It always shows:

'Attempting to establish a Remote Desktop connection with Windows... Failed!

ERROR: REMOTE DESKTOP PROTOCOL FAILURE. FreeRDP failed to establish a connection with Windows.'

I find it hard to believe that the distro used for containers cannot be used with WinApps.

Help, please."

**Comment by semedilino-073** [comment]
For anyone still wondering, this was the fix for me:
https://gist.github.com/TheEEs/ebae8423a4b4fae7cb9531830b62d186
It is a customized version of setup.sh to run on Alpine systems
